README.md

```diff
@@ -125,6 +125,7 @@ The item in `disks` list of `profile` dictionary can contain following attribute
 |--------------------|----------------|----------------------------------------------|
 | size               | UNDEF          | The size of the additional disk. |
 | name               | UNDEF          | The name of the additional disk.  |
+| id               | UNDEF          | The id of the disk which you want to update. Also used for renaming of disk.  |
```

Comment:
Please write here: "Id of the disk. If you pass <i>id</id> of the disk and <i>name</i> the disk will be looked up by <i>id</id> and will update name of the disk if it differs from the name passed in <i>name</i> parameter."